# Patch for 0.9.17 to adjust configuration
# - Adjust session manager logging configuration
diff -urNad xrdp-0.9.17-orig/sesman/sesman.ini xrdp-0.9.17/sesman/sesman.ini
--- xrdp-0.9.17-orig/sesman/sesman.ini	2021-09-01 04:32:12.000000000 +0300
+++ xrdp-0.9.17/sesman/sesman.ini	2021-09-29 20:39:13.063609032 +0300
@@ -12,7 +12,7 @@
 ReconnectScript=reconnectwm.sh
 
 [Security]
-AllowRootLogin=true
+AllowRootLogin=false
 MaxLoginRetry=4
 TerminalServerUsers=tsusers
 TerminalServerAdmins=tsadmins
@@ -134,7 +134,7 @@
 ; Note: Log levels can be any of: core, error, warning, info, debug, or trace
 LogLevel=INFO
 EnableSyslog=true
-#SyslogLevel=INFO
+SyslogLevel=INFO
 #EnableConsole=false
 #ConsoleLevel=INFO
 #EnableProcessId=false
diff -urNad xrdp-0.9.17-orig/xrdp/xrdp.ini xrdp-0.9.17/xrdp/xrdp.ini
--- xrdp-0.9.17-orig/xrdp/xrdp.ini	2021-09-01 04:32:12.000000000 +0300
+++ xrdp-0.9.17/xrdp/xrdp.ini	2021-09-29 20:39:13.147609294 +0300
@@ -166,7 +166,7 @@
 LogFile=xrdp.log
 LogLevel=INFO
 EnableSyslog=true
-#SyslogLevel=INFO
+SyslogLevel=INFO
 #EnableConsole=false
 #ConsoleLevel=INFO
 #EnableProcessId=false
